.azure-pipelines/scripts/datadog_checks_base/windows/55_vcpython27.bat

```diff
@@ -0,0 +1 @@
+choco install vcpython27 --yes
```

Comment:
can you add comments on this file?
Is to leave this comment in somewhere:
```
# Need to install Microsoft Visuall C++ 9.0 for python 2.7 manually because of
# https://github.com/microsoft/azure-pipelines-image-generation/issues/793/
# Some dependencies require compilation steps, at time of writing only `mmh3`.
```